src/shared/Inputs/index.jsx

```diff
@@ -1,6 +1,11 @@
-import React, { useEffect, useState, useRef, useCallback } from 'react';
+import React, {
+  useEffect, useState, useRef, useCallback,
+} from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { TextField, Radio, RadioGroup, FormControlLabel, FormLabel, Autocomplete as MUIAutocomplete } from '@mui/material';
+import {
+  TextField, Radio, RadioGroup, FormControlLabel, FormLabel, Autocomplete as MUIAutocomplete,
```

Comment:
should this use PSARadiobutton?

Reply:
This component is replaced in the user history pr and will not be used anymore